DXRCore/DeusEx/Classes/DXRInfo.uc

```diff
@@ -237,6 +237,14 @@ function bool OnTitleScreen()
     return dxr.LocalURL=="DX" || dxr.LocalURL=="DXONLY";
 }
 
+function bool OnEndgameMap()
+{
+    local DXRando dxr;
+    dxr = class'DXRando'.default.dxr;
+    if(dxr == None) return true;
+    return dxr.dxInfo.missionNumber == 99;
```

Comment:
doesn't this fail to detect dance party ending? also the function should maybe be called `IsEndgameMap()`